src/frontend-samples/explode-sample/ExplodeApp.tsx

```diff
@@ -0,0 +1,337 @@
+      };
+      loadTile().then(tileLoadedCallback);
+    });
+  }
+
+  /** This method calculates and populates the transform for diplacing the "exploded" elements. */
+  private static populateExplosionTransform(elements: ElementData[], explosionFactor: number) {
+    // Find the center of the range containing all element
+    // Have to do it after the world transform is populated.
+    const center = getRangeUnion(elements).center;
+    // Create transform for "exploding" effect
+    elements.forEach((element) => {
+      const vector = Vector3d.createFrom(center);
+      vector.subtractInPlace(element.boundingBox.center);
+      vector.scaleInPlace(explosionFactor);
+      element.transformExplode = Transform.createTranslation(vector);
+    });
+  }
+
+  /** Queries the backend for the elements to explode.  It also populates the data it can interpolate with a single pass. */
+  private static async queryElements(vp: Viewport, elementsIds: string[]): Promise<ElementData[]> {
+    const query = `Select ECInstanceID,Origin,Yaw,Pitch,Roll,BBoxLow,BBoxHigh FROM BisCore:GeometricElement3d WHERE ECInstanceID IN (${elementsIds.join(",")})`;
+    const results = vp.iModel.query(query);
+    const data: ElementData[] = [];
+    let row = await results.next();
+    while (row.value) {
+      const element = (row.value as ElementData);
+      element.isLoaded = false;
+
+      // Populate information from first pass
+      const placement = Placement3d.fromJSON({ origin: element.origin, angles: { pitch: element.pitch, roll: element.roll, yaw: element.yaw } });
+      const transform = placement.transform;
+      const box = Range3d.create(element.bBoxHigh, element.bBoxLow);
+
+      element.transformWorld = transform;
+      element.boundingBox = transform.multiplyRange(box);
+
+      data.push(element);
+      row = await results.next();
+    }
+
+    return data;
+  }
+
+  /** Reads the response for tile content and returns the Tile Content. */
+  private static async readResponse(vp: Viewport, response: TileRequest.ResponseData): Promise<TileContent> {
+    const stream = new ByteStream((response as Uint8Array).buffer);
+    const reader = ImdlReader.create(stream, vp.iModel, vp.iModel.iModelId!, vp.view.is3d(), IModelApp.renderSystem);
+
+    let content: TileContent = { isLeaf: true };
+    if (reader) {
+      try {
+        content = await reader.read();
+      } catch (_) {
+        //
+      }
+    }
+    return content;
+  }
+}
+
+/** This provider both hides the original graphics of the element and inserts the transformed graphics. */
+class ExplodeProvider implements TiledGraphicsProvider, FeatureOverrideProvider {
+  private static _instances = new Map<number, ExplodeProvider>();
+  public data: ElementData[] = [];
+  /** Returns a provider associated with a given viewport. If one does not exist, it will be created. */
+  public static getOrCreate(vp: Viewport) {
+    let provider = ExplodeProvider._instances.get(vp.viewportId);
```

Comment:
Get rid of your map. Use `viewport.findFeatureOverrideProviderOfType<ExplodeProvider>()` instead.